### PR TITLE
COMP: update download location for pre-built ICU for Visual Studio

### PR DESCRIPTION
**Pull request by dzenanz**
This is required to build DCMTK with ICU enabled, as MIDAS server is no longer working.

### CODE REVIEW

**Comment by dzenanz** [comment]
@hjmjohnson the solution to this problem turned out to be to find the missing files on the new server (data.kitware.com).

File: Modules/ThirdParty/DCMTK/CMakeLists.txt [hunk 1]
```diff
@@ -52,13 +52,13 @@ if(DCMTK_USE_ICU)
...
-      set(ICU_1911_64 462048 1706884a6fb773caad3f3955747cf7db)
+      # Create map of SHA512/MD5 hashes
+      set(ICU_1800_32 9b134c1fe8f5fcc3afe34f6a72d4c9fded776dfd33d7073861c2de32c40d4bda15ca9eb32613c262b13b479bf88bdd34658cee5505a1d9b7e3779104fd9440d3 39c597cf28f2df71baf92156d9276a4e)
+      set(ICU_1800_64 023c9f1dc2469058aa4c0933d82cfdab37fc803dc86f8b936f2001f0b600ba02f8c42b695398d3ed88def6feec6b5e01561d86377a3fc582788bc35c4f9d912b 9f9bc9e165ce47182889a748c20028c3)
+      set(ICU_1900_32 75cc1bb4ab6fce2df77cacb80e86be6e45d1a3bc0a7f96d5fd7193b0ddd67a2094d97626a2c59888b4374efdba7381ed3598333237b25b8bcce4777ba76731d7 86f191c160ff86e66437c23ae36c50f8)
+      set(ICU_1900_64 2b200cb120c32ead26ce94aea1d531754f3c530fcb2cd26f9bf3b576c040009fdf2504376b843257a70223c8d8331359540e2ac6587004eaf9e5f7941bcf22b8 3db188b9ba24ca39a0c5d1411b591d4d)
+      set(ICU_1911_32 b69569454639452c6636bc19d5e75ded119cf9a6f1f3fa8344662b481fdf7574d5ded54e7f36c69b3c752b4a1c3ea21597ae569376a888f030e6ee8d64268d0e e4053721d7d11125d82df34e0a576c2a)
+      set(ICU_1911_64 462048 1706884a6fb773caad3f3955747cf7db) # this was not migrated to Girder instance at data.kitware.com
```
Comment (hjmjohnson) [inline Modules/ThirdParty/DCMTK/CMakeLists.txt]: Is the comment useful in the codebase?  It seems that you have fixed this.

**Comment by dzenanz** [comment]
@hjmjohnson the solution to this problem turned out to be to find the missing files on the new server (data.kitware.com).

**Comment by hjmjohnson** [comment]
@dzenanz Looks good in principle.  Some CI failures.

**Comment by dzenanz** [comment]
The CI failures are unrelated (HDF5).

**Pull request merged**